VerveAdapter/src/main/java/com/chartboost/mediation/verveadapter/VerveAdapter.kt

```diff
@@ -165,9 +165,9 @@ class VerveAdapter : PartnerAdapter {
         PartnerLogController.log(BIDDER_INFO_FETCH_STARTED)
 
         return withContext(Dispatchers.IO) {
-            val token = HyBid.getAppToken() ?: ""
-            PartnerLogController.log(if (token.isEmpty()) BIDDER_INFO_FETCH_FAILED else BIDDER_INFO_FETCH_SUCCEEDED)
-            mapOf("app_auth_token" to token)
+            val signalData = HyBid.getCustomRequestSignalData("cb") ?: ""
+            PartnerLogController.log(if (signalData.isEmpty()) BIDDER_INFO_FETCH_FAILED else BIDDER_INFO_FETCH_SUCCEEDED)
+            mapOf("signal_data" to signalData)
```

Comment:
I believe `app_auth_token`/`signal_data` is intended for our backend to parse. If so, we cannot just change this here since backend would be expecting a different key name (i.e. `app_auth_token`). We'll need to check on this internally.

Reply:
Agreed here @chauduyphanvu . I've reached out in parallel to our contact in your team. Since we'll need to sync on how to send this to our backend. Since this is what it is expecting to extract some extra params for the bid 🙂

Reply:
We can double-check with our backend. 🤔